src/views/User/components/WelcomeDialog/WelcomeDialog.js

```diff
@@ -53,3 +53,8 @@ const WelcomeDialog = props => {
 }
 
 export default WelcomeDialog
+
+WelcomeDialog.propTypes = {
+	setDialogVisible: PropTypes.func.isRequired,
```

Comment:
why do you add `setDialogVisible `in propTypes without using `setDialogVisible `variable in this component?